android/SmartcarMqttController/app/src/main/java/platis/solutions/smartcarmqttcontroller/Plants.java

```diff
@@ -0,0 +1,61 @@
+package platis.solutions.smartcarmqttcontroller;
+
+public class Plants {
+
+    private String ID;
+    private String species;
+    private Integer row;
+    private Integer column;
+    private String health;
+
+    public Plants(){} // needed for later
+
+    public Plants (String ID,String species,int row, int column, String health ){
+        this.ID = ID;
+        this.species = species;
+        this.row = row;
+        this.column = column;
+        this.health = health;
+
+    }
+
+    public String getID() {
+        return ID;
+    }
+
+    public void setID(String ID) {
+        this.ID = ID;
+    }
+
+    public String getSpecies() {
+        return species;
+    }
+
+    public void setSpecies(String species) {
+        this.species = species;
+    }
+
+    public Integer getRow() {
+        return row;
+    }
+
+    public void setRow(Integer row) {
+        this.row = row;
+    }
+
+    public Integer getColumn() {
+        return column;
+    }
+
+    public void setColumn(Integer column) {
+        this.column = column;
+    }
+
+    public String getHealth() {
+        return health;
+    }
+
+    public void setHealth(String health) {
+        this.health = health;
+    }
```

Comment:
I'm curious, unless you are planning to make these more complex, why not remove all of these trivial setters and getters and simply make the members public?

Reply:
Hi, we will need most of them later and want to make sure it is encapsulated.

Reply:
I wouldn't say the **trivial** setters and getters, which have exactly the same names as the member variables (and thus exposing internal implementation details) and do not do any sanity checks on the input or transformations on the output are encapsulating anything, but I will let you think of it more... ;)

Reply:
Record types should not be encapsulated; since Java 14, there is even language support for records!
(cf. https://docs.oracle.com/en/java/javase/14/language/records.html)